### PR TITLE
feat: rtx for estates

### PR DESCRIPTION
**Pull request by nachomazzara**

### CODE REVIEW

File: webapp/src/modules/transaction/sagas.js [hunk 1]
```diff
@@ -36,7 +36,8 @@ function* handleTransactionRequest(action = {}) {
     yield put(
       fetchTransactionSuccess({
         ...transaction,
-        status: TRANSACTION_STATUS.confirmed
+        status: TRANSACTION_STATUS.confirmed,
+        receipt
```
Comment (cazala) [inline webapp/src/modules/transaction/sagas.js]: note to self: we need to add this to `decentraland-dapps`

Comment (nachomazzara) [reply webapp/src/modules/transaction/sagas.js]: @cazala  Do you want me to create an issue?

Comment (cazala) [reply webapp/src/modules/transaction/sagas.js]: https://github.com/decentraland/decentraland-dapps/issues/8

File: webapp/src/modules/address/reducer.js [hunk 2]
```diff
@@ -153,6 +161,97 @@ export function addressReducer(state = INITIAL_STATE, action) {
...
+            data: {
+              ...state.data,
+              [transaction.from]: {
+                ...user,
+                parcel_ids: updatedParcelIds
+              }
+            }
+          }
+        }
+        case TRANSFER_ESTATE_SUCCESS: {
```
Comment (cazala) [inline webapp/src/modules/address/reducer.js]: This is removing the estate from the old owner but it's not adding it to the new owner. Check TRANSFER_PARCEL_SUCCESS reference implementation:

```js
case TRANSFER_PARCEL_SUCCESS: {
          const { oldOwner, newOwner, x, y } = transaction.payload
          const parcelId = buildCoordinate(x, y)
          const oldOwnerAddress = state.data[oldOwner] || { ...EMPTY_ADDRESS }
          const newOwnerAddress = state.data[newOwner] || { ...EMPTY_ADDRESS }
          return {
            ...state,
            data: {
              ...state.data,
              [oldOwner]: {
                ...oldOwnerAddress,
                parcel_ids: oldOwnerAddress.parcel_ids.filter(
                  x => x !== parcelId
                )
              },
              [newOwner]: {
                ...newOwnerAddress,
                parcel_ids: newOwnerAddress.parcel_ids.concat(parcelId)
              }
            }
          }
        }
```

File: webapp/src/modules/address/reducer.js [hunk 3]
```diff
@@ -153,6 +161,97 @@ export function addressReducer(state = INITIAL_STATE, action) {
...
+              [transaction.from]: {
+                ...user,
+                estate_ids: user.estate_ids.filter(
+                  estateId => estateId !== estate.asset_id
+                )
+              }
+            }
+          }
+        }
+        case DELETE_ESTATE_SUCCESS: {
```
Comment (cazala) [inline webapp/src/modules/address/reducer.js]: This is adding the parcels from the removed Estate to the address, but it's not removing the Estate (so after executing this case the address has both the Estate and the parcels). We need to remove the estate_id from this address's `estate_ids`

File: webapp/src/modules/parcels/reducer.js [hunk 4]
```diff
@@ -253,6 +261,78 @@ export function parcelsReducer(state = INITIAL_STATE, action) {
...
+          estate.data.parcels.forEach(parcel => {
+            const updatedParcel = updatedParcels.find(
+              p => p.x === parcel.x && p.y === parcel.y
+            )
+            if (!updatedParcel) {
+              const parcelId = buildCoordinate(parcel.x, parcel.y)
+              updatedParcels.push(state.data[parcelId])
+            }
+          })
+          console.log(updatedParcels)
```
Comment (cazala) [inline webapp/src/modules/parcels/reducer.js]: oops

File: webapp/src/modules/address/reducer.js (hunk 3 above)
Comment (nachomazzara) [reply webapp/src/modules/address/reducer.js]: We are supporting behind empty estates (the server returns them) but we are not showing them at front.

**Pull request merged**